docs/tutorials/FlowTutorial.md

```diff
@@ -231,6 +231,14 @@ make DESIGN_CONFIG=./designs/sky130hd/ibex/config.mk
 As the flow executes, check out the ORFS directory contents and their
 significance.
 
+If you started running a design and the terminal in which you initiated the run was closed for some reason, you may want to delete all partially generated files and start a fresh run(`ibex` in this case). You can accomplish this task by:
```

Comment:
ORFS can generally restart from a previous partial run.  You would only need to do this if you had a failure in the middle of writing some file.  I think the description needs to make this clear.

Reply:
Sure sir, I shall make the required changes.